src/frontend/cellprofiler/modules/morph.py

```diff
@@ -310,9 +311,9 @@
     F_VBREAK,
 ]
 
-R_ONCE = "Once"
-R_FOREVER = "Forever"
-R_CUSTOM = "Custom"
+R_ONCE = RepeatMethod.ONCE
+R_FOREVER = RepeatMethod.FOREVER
+R_CUSTOM = RepeatMethod.CUSTOM
```

Comment:
Let's remove this from library opts, and keep them frontend only. They don't affect library behaviour, and library never sees these values.